sass/_overlay.scss

```diff
@@ -5,6 +5,7 @@ Overlay for menus and widgets
 .overlay {
 	@include rem(font-size, 20px);
     background: $overlay-color;
+    color: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.